docs/zkapps/advanced-snarkyjs/custom-tokens.mdx

```diff
@@ -121,8 +214,8 @@ Suppose a token account is being created for the first time. In that case, an ac
 
 Aside from sending custom tokens, custom tokens can be minted and burned by a **token owner account**. A token owner account is the governing zkApp account for a specific custom token.
 
-#### Token Owner
+### Token Owner
 
 A token owner is an account that creates, facilitates, and governs how a custom token is to be used. Concretely, the token owner is the account that created the custom token and is the only account that can mint and burn tokens.
 
-In addition to being the only account that can mint and burn tokens, the token owner is also the only account that can authorize sending tokens between two accounts. If two accounts want to send tokens to each other, the token owner must authorize the transaction. The token owner can authorize the transaction either with a signature or proof.
+In addition to being the only account that can mint and burn tokens, the token owner is also the only account that can approve sending tokens between two accounts. If two accounts want to send tokens to each other, the token owner must approve the transaction. The token owner can approve the transaction either with a signature or proof.
```

Comment:
You're using the world `approve` in this paragraph, but `Signature Authorization` and `Proof Authorization` above. Intentional?